google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright 2020 Google LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.misc;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v3.enums.MediaTypeEnum.MediaType;
+import com.google.ads.googleads.v3.errors.GoogleAdsError;
+import com.google.ads.googleads.v3.errors.GoogleAdsException;
+import com.google.ads.googleads.v3.resources.MediaBundle;
+import com.google.ads.googleads.v3.resources.MediaFile;
+import com.google.ads.googleads.v3.services.MediaFileOperation;
+import com.google.ads.googleads.v3.services.MediaFileServiceClient;
+import com.google.ads.googleads.v3.services.MutateMediaFilesResponse;
+import com.google.common.io.ByteStreams;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.BytesValue;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+
+/**
+ * This example uploads an HTML5 zip file as a Media Bundle. More information about Media Bundle can
```

Comment:
Can abbreviate the start of this sentence to "Uploads an HTML5..."

Reply:
NP